hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1743,7 +1745,16 @@ message ListSnapshotResponse {
 }
 
 message SnapshotDiffResponse {
+  enum JobStatusProto {
+    QUEUED = 1;
+    IN_PROGRESS = 2;
+    DONE = 3;
+    REJECTED = 4;
+    FAILED = 5;
+  }
```

Comment:
This occurred to me: What would be the response status to a SnapDiff request (B) that already has a same SnapDiff request (A) that is `IN_PROGRESS`? B should immediately get `IN_PROGRESS` right?